packages/frontend/web-ui/src/app/store/helpers/createInitialState.spec.ts

```diff
@@ -1,12 +1,13 @@
 import { afterAll, beforeAll, describe, expect, it, jest } from '@jest/globals';
 import { createInitialState } from './createInitialState';
+import { AuthState } from './models/AuthState';
 import {
   STATUS_AUTH_AUTHENTICATED,
   STATUS_AUTH_NOT_AUTHENTICATED,
 } from '../data/authSliceStatus';
 
 describe(createInitialState.name, () => {
-  let getItemSpy;
+  let getItemSpy: jest.SpiedFunction<any>;
```

Comment:
For now, let's avoid using `any`. Let's use a better type instead:

```suggestion
  let getItemSpy: jest.SpiedFunction<typeof window.localStorage.getItem>;
```

Reply:
I didn't remember the key word `typeof` but now, I understand completely!